src/typings/en.translation.d.ts

```diff
@@ -11,10 +11,6 @@ declare namespace typed_i18n {
   readonly "searchBook": string,
   readonly "skipLink": string
 };
-  (_: "accessibility.bookAuthor", __?: {}): string;
-  (_: "accessibility.bookPublisher", __?: {}): string;
-  (_: "accessibility.bookReleaseDate", __?: {}): string;
-  (_: "accessibility.bookTitle", __?: {}): string;
```

Comment:
I see that the `readonly "bookAuthor": string,` (etc.) lines are still present in this file, which indicates that this code removal was done manually. In order to maintain locales, I would strongly recommend using the automated scripts instead: `npm run i18n:typed` (the other commands perform other tasks `npm run i18n:sort` and `npm run i18n:scan`). See README:
https://github.com/readium/readium-desktop/blob/develop/README.md#localization--ui-translations

Reply:
Oh, I didn't know there was a script, it's easier like that